Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -57,9 +59,27 @@ public async Task PrintResponses(string commandArgument, Contexts contexts)
         private IEnumerable<KeyValuePair<string, string>> GetAllResponses()
         {
             return this._responsesDatabase.GetResponsesFromBase()
-                .Select(x => new KeyValuePair<string, string>(x.OnEvent, this.GetRawMessage(x.Message)));
+                .Select(x => new KeyValuePair<string, string>(x.OnEvent, this.GetResponseWithVariableList(x)));
         }
 
+        private string GetResponseWithVariableList(Response response)
+        {
+            var sb = new StringBuilder("\n__Dostępne zmienne:__");
```

Comment:
staraj się unikać skrótów w nazwach zmiennych
później jak jest dużo zmiennych to robi sie syf

Reply:
Ok, ale chyba logiczne że sb to StringBuilder? No chyba że tylko w Javie ten obiekt jest tak często wykorzystywany

Reply:
`ale chyba logiczne że sb to StringBuilder`
no jeśli wszędzie w projekcie mamy `stringBuilder` to tak niezbyt 
jeśli każdy zacznie pisać w swoim stylu, to po chwili kod będzie nieczytelny dla wszystkich

Reply:
zmieniłem